lib/api/helpers.js

```diff
@@ -4,12 +4,22 @@ import { FlowRouter } from "meteor/kadira:flow-router-ssr";
 import { Shops } from "/lib/collections";
 
 /* eslint no-unused-vars: 0 */
-//
-//  TODO review this slugify import in lib/api/helpers
-//
+
+// Export Reaction using commonJS style for common libraries to use Reaction.*
+// https://docs.meteor.com/packages/modules.html#CommonJS
+let Reaction;
```

Comment:
We need a common Reaction object for common libs to use safely. 

This is only a suggestion and we'd like your feedback on this to see if it suits everyones needs.
Meteor suggests to use commonJS require() style as the only way to include modules conditionally (see page for more)